README.md

```diff
@@ -542,7 +599,7 @@ The format of this input file should follow the [Two Column Format](two-column-f
 To plot the scurves, and their fits, for VFAT0 channel 29 from a set of scandates defined in `listOfScanDates_Scurve.txt` taken by `ultraScurve.py` and analyzed with `anaUltraScurve.py` you would call:
 
 ```
-gemSCurveAnaToolkit.py -ilistOfScanDates_Scurve.txt -v0 -s29 --anaType=scurveAna -c --summary --drawLeg 
+gemSCurveAnaToolkit.py -ilistOfScanDates_Scurve.txt -v0 -s29 --anaType=scurveAna -c --summary --drawLeg
```

Comment:
I cannot see what changed here...?

Reply:
Turned `U+2028 LINE SEPARATOR` into an ASCII newline (not that I was aware of it... but remember I had an issue with a spurious newline here).

Since it's the only place where this character was present, I would say this is a fix. I can reintroduce it, but not every tool may work well with such an exotic character.

*\<troll>If you ask me to put it back, next time I may introduce other funny characters such as ͘ı orａ\</troll>*

Reply:
No I was just trying to understand.  Evidently GitHub is just not rendering the character in question.  Proceed with other changes as discussed above.  Thanks.